WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
Maybe it's just me, but the name seems a bit misleading. My first thought would be `:next_round_cutoff` -> *Some kind of cutoff that will be applied during the next round.*